tsconfig.json

```diff
@@ -1,29 +1,26 @@
 {
+    "files": [
+        "index.ts"
+    ],
     "compilerOptions": {
         "target": "es5",
-        "module": "umd",
+        "module": "es2015",
+        "lib": ["es6", "es2015", "dom"],
         "moduleResolution": "node",
-        "isolatedModules": false,
-        "jsx": "react",
         "experimentalDecorators": true,
         "emitDecoratorMetadata": true,
-        "declaration": false,
+        "declaration": true,
         "noImplicitAny": true,
-        "noImplicitUseStrict": false,
         "removeComments": true,
-        "noLib": false,
-        "preserveConstEnums": true,
         "suppressImplicitAnyIndexErrors": true,
-        "sourceMap": true,
+        "inlineSourceMap": true,
```

Comment:
That's a vigilant eye! :)

No particular reason. Thought I'll be less files here and for the diff. You want me to revert this one?